packages/analytics/analytics-utilities/src/types/explore-v4.ts

```diff
@@ -159,7 +196,7 @@ export interface QueryResponseMeta {
   start_ms: number
   end_ms: number
   display: DisplayBlob
-  metric_names?: ExploreAggregations[]
+  metric_names?: ExploreAggregations[] | AiExploreAggregations[]
```

Comment:
I think this should be `AllAggregations[]`.  Otherwise, I think it's looking good...